lib/aptible/cli/subcommands/restart.rb

```diff
@@ -8,10 +8,34 @@ def self.included(thor)
             include Helpers::App
 
             desc 'restart', 'Restart all services associated with an app'
+            option :'simulate-oom',
```

Comment:
Thanks @fancyremarker; didn't realize we could use `simulate_oom` directly and have the Thor handle the hyphen :)